cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonLanguageFrontend.kt

```diff
@@ -435,6 +435,7 @@ fun fromPython(pyObject: Any?): Python.BaseObject {
             "ast.arg" -> Python.AST.arg(pyObject)
             "ast.arguments" -> Python.AST.arguments(pyObject)
             "ast.comprehension" -> Python.AST.comprehension(pyObject)
+            "ast.ExceptHandler",
```

Comment:
Is this intentional?

Reply:
I have no idea, it looks like it was intentional, maybe on a different version of Python the name of the ast node is different. We had such an issue before.

Reply:
Calling the python AST doctor @maximiliankaul

Reply:
See #1730

Reply:
Should be resolved with the other PR.